pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -84,6 +84,13 @@ func (e *InvalidRegistries) Error() string {
 	return e.s
 }
 
+// ValidatePrefix calls parseURL but discards the normalized registry value returned
+// It just checks that the registry value given as an input is valid
```

Comment:
This _should_ be more precise, though, so that it is used correctly in callers: it’s a registry[/namespace…[/repo]], valid values for the `Registry.{Prefix,URL}` and `Mirror.URL` fields.

(@vrothberg without a digest / :tag suffix, right?)